airflow/contrib/operators/qubole_operator.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import datetime
+import os
+
+from qds_sdk.qubole import Qubole
+from qds_sdk.commands import *
```

Comment:
`import *` is consider bad practice because it makes it impossible to track which namespace objects are coming from.